tech/languages/java/java-installation.md

```diff
@@ -8,34 +8,26 @@ description: General-purpose, object-oriented and concurrent computer programmin
 
 # Java installation
 
-Fedora comes with [OpenJDK](http://openjdk.java.net/) - a free and open source implementation of the Java Platform, Standard Edition. To install it, simply type:
+Fedora comes with [OpenJDK](https://openjdk.java.net/) - a free and open source implementation of the Java Platform, Standard Edition. To install it, simply type:
 
 ```
-$ sudo dnf install java-15-openjdk-devel
```

Comment:
Hmm, I have to use `sudo dnf install java-17-openjdk-devel` to install the devel package as the `sudo dnf install java-openjdk-devel` cannot find the proper package. Is this expected?

Reply:
Thanks for catching this. This is not expected to me, but I will open a bug to let OpenJDK maintainers clarify that.
In meantime will update this PR to use other generic (unversioned) forms of package names that are available.

Reply:
Bugzilla bug opened: https://bugzilla.redhat.com/show_bug.cgi?id=2084332

Reply:
Filed a reminder for us here too: https://github.com/developer-portal/content/issues/435